### ISSUE TITLE
MaskedInput -- Should announce when masked options drop is open

### ISSUE THREAD

**Issue by taysea** [grommet/grommet#7573]
The "options" for MaskedInput mask sections should behave more closely to TextInput suggestions with `comboxBoxProps`. This will help inform screen reader users about the presences of suggestions for each masked section.

Related to WCAG 1.3.1

**Comment by jcfilben** [comment]
@taysea I just merged #7588 as a follow on can we update the WCAG rule status for MaskedInput 1.3.1 on the site
